assets/airports/kpvd.json

```diff
@@ -0,0 +1,6536 @@
+{
+    "radio": {
+        "twr": "Providence Tower",
+        "app": "Providence Approach",
+        "dep": "Providence Departure"
+    },
+    "icao": "KPVD",
+    "iata": "PVD",
+    "magnetic_north": -14.5,
+    "ctr_radius": 80,
+    "ctr_ceiling": 12000,
+    "initial_alt": 4000,
+    "position": ["N41d43m44.00", "W071d25m69.00", "54ft"],
+    "rr_radius_nm": 5.0,
+    "rr_center": ["N41d43m44.00", "W071d25m69.00"],
+    "has_terrain": true,
+    "wind": {
+        "angle": 310,
+        "speed": 9
+    },
+    "arrivalRunway": "23",
+    "departureRunway": "23",
+    "airspace": [
+        {
+            "floor": 0,
+            "ceiling": 120,
+            "airspace_class": "C",
+            "poly": [
+                ["N041d53m40.00",  "W070d55m29.00"],
+                ["N041d54m21.00",  "W070d53m00.00"],
+                ["N041d47m59.00",  "W070d42m00.00"],
+                ["N041d42m09.00",  "W070d47m14.00"],
+                ["N041d33m50.00",  "W070d49m59.00"],
+                ["N041d26m14.00",  "W071d01m49.00"],
+                ["N040d57m49.00",  "W071d01m46.00"],
+                ["N040d45m53.00",  "W071d51m09.00"],
+                ["N041d03m53.00",  "W072d10m26.00"],
+                ["N041d07m30.00",  "W072d15m00.00"],
+                ["N041d21m44.00",  "W072d23m00.00"],
+                ["N041d29m00.00",  "W072d28m00.00"],
+                ["N041d36m24.00",  "W072d09m15.00"],
+                ["N041d48m09.00",  "W072d03m45.00"],
+                ["N041d59m44.00",  "W071d55m29.00"],
+                ["N042d06m00.00",  "W071d29m00.00"],
+                ["N042d00m56.00",  "W071d21m38.00"],
+                ["N041d55m01.00",  "W071d13m13.00"]
+            ]
+        }
+    ],
+    "fixes": {
```

Comment:
There seem to be a lot of unnecessary fixes here and it almost looks as crowded as LOWW did. I'd remove most of them and just keep the ones in use and maybe some other ones that you consider important (VOR's, ILS fixes, etc.)